### DISCUSSION TITLE
Add fluid model at a specific time

### DISCUSSION THREAD

**Discussion by Yangaoxu** [InteractiveComputerGraphics/SPlisHSPlasH#258] [Q&A] [answered]
If I want to add certain FluidModels after the scene starts running for a certain time, how should I configure scene files? I know Emitters can do this, but what if I want to add any fluid model at any time in a simulation? If anyone answers, thank you very much.

**Comment by janbender** [comment, marked as answer]
At the moment such a function is not implemented. However, there are several ways to do this. First, you could implement a sepcial emitter which adds your models at specific times. Second, you could add the fluid model at the beginning of the simulation but set all particles to inactive and maybe translate the particles to a position where they do not disturb. At a desired point in time translate the particles back and activate them. This could be done via a Python script.

**Reply by Yangaoxu** [reply]
It's very useful, thank you so much!
